app/Http/Controllers/Customer/InvoiceController.php

```diff
@@ -438,6 +438,9 @@ public function enable(Invoice $invoice) {
                 SystemMessageService::addWarningMessage("system_messages.invoice.is_empty");
         }
 
+        $newCreatedAt = now();
```

Comment:
It is better to avoid creating unnecessary variables as it makes the code slightly more readable